docs/general/ad-filtering/create-own-filters.md

```diff
@@ -740,6 +759,28 @@ We recommend to get acquainted with [this article](https://adblockplus.org/filte
 
 :::
 
+<!-- Please keep them sorted -->
+
+| Modifier \ Products | [CL apps][cl-apps] | [ExtChr][ext-chr] | [ExtFf][ext-ff] | [iOS app][ios-app] | [Safari][ext-saf] | [ExtEdg][ext-edg] | [CB][and-cb] |
+| --- | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
+| [$content](#content-modifier) | ✅ | ❌ | ✅ | ❌ | ❌ | ❌ | ❌ |
+| [$elemhide](#elemhide-modifier) | ✅ | ✅ | ✅ | ? | ? | ✅ | ? |
```

Comment:
* `$elemhide`, `$generichide`, '$jsinject' works in Safari/iOS
* `$urlblock` / `$genericblock` works as `$document` exclusion in Safari/iOS. It is suported by apps and chromium/ff extensions.

* `$elemhide`, `$generichide` work in `CB` as it should. 
* * `$urlblock` / `$genericblock` don't work in `CB`

Reply:
unblocks everything